logical/src/test/java/org/apache/drill/common/expression/fn/JodaDateValidatorTest.java

```diff
@@ -24,6 +24,7 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import java.util.Locale;
```

Comment:
Could you please revert changes, made in this class to the version, where you just added `.withLocale(Locale.US)` in `parseDateFromPostgres` method. My earlier comment was connected only with change, made in UDF function, but here, in the method that just used for unit tests, we can do that.